test/table2qb/configuration/column_test.clj

```diff
@@ -2,58 +2,38 @@
   (:require [clojure.test :refer :all]
             [table2qb.configuration.column :refer :all]))
 
```

Comment:
I wonder if it makes sense to add some examples to `columns-test`? We have unit tests in `column-test` for each validation, and `csv-test` checks that the parser works but we're not checking that we've specified the columns input correctly.

Reply:
Yes that would be useful!

Reply:
Done in 689660c.